src/theme/custom.css

```
@@ -77,3 +77,9 @@ th {
 .segment-wrapper {
   padding: 8px 16px;
 }
+@media (min-width:961px) {
```

I would use the same values for `min-width` and `min-height` as in the CSS from Ionic:

- https://github.com/ionic-team/ionic/blob/e3364f283f5660e4325b5d36e552824a9a24839d/core/src/components/modal/modal.scss#L78
- https://github.com/ionic-team/ionic/blob/e3364f283f5660e4325b5d36e552824a9a24839d/core/src/components/modal/modal.vars.scss#L7

```suggestion

/* Modal Wrapper - Overwrite modal size for large screens. */
@media only screen and (min-width: 768px) and (min-height: 600px) {
```

---

I have updated this as per your request